Changing version to 3.0.1 (#10)
okta/okta-sdk-abstractions-dotnet: CHANGELOG.md

# Changelog
Running changelog of releases since `2.0.1`

## v3.0.1

> - In `UserAgentBuilder` added sanitization of the Runtime Version token. Bracket symbols will be replaced with dashes on sanitization.  

Reviewer: ```suggestion
- Sanitize runtime version token in `UserAgentBuilder`. Bracket symbols are replaced with dashes on sanitization.  
```
